src/execution_plan/ops/op_expand_into.c

```diff
@@ -158,15 +158,16 @@ static Record _handoff(OpExpandInto *op) {
      * nodes are connected. */
     while(op->recordsLen) {
         op->recordsLen--;
+        // Current record resides at row recordsLen.
+        int rowIdx = op->recordsLen;
```

Comment:
I think this struct member should be named `recordCount` for clarity.

Reply:
👍